package.json

```
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "webpack": "webpack",
-    "package": "source .env && echo PPK=$PPK && kintone-plugin-packer src --ppk $PPK",
+    "package": "kintone-plugin-packer src --ppk myppk.ppk",
```

```suggestion
    "source .env && echo PPK=$PPK && kintone-plugin-packer src --ppk $PPK",
```
修正前の方は動きます💦
元に戻していただけますでしょうか。